.tflint.hcl

```diff
@@ -0,0 +1,20 @@
+tflint {
+  required_version = ">= 0.52"
+  preset           = "recommended"
+}
+
+rule "terraform_naming_convention" {
+  enabled = true
+}
+
+rule "terraform_required_version" {
+  enabled = false
+}
+
+rule "terraform_required_providers" {
+  enabled = false
+}
+
+rule "terraform_unused_declarations" {
```

Comment:
Why is it disabled?

Reply:
some of data sources in our examples is unused (has no output)